app/views/summary/index.html.haml

```diff
@@ -0,0 +1,43 @@
+%div.inset
+  %table.table.table-striped.table-bordered.table-responsive
+    %caption
+      %span.namespace= @summary.client_namespace
+      Requests: Fiscal Year
+      = @summary.fiscal_year
```

Comment:
if I am understanding the code correctly, of someone visited this page at `/summary`it would show results for the current year but `summary/2015